target/linux/ipq40xx/files-6.12/arch/arm/boot/dts/qcom/qcom-ipq4018-a42.dts

```diff
@@ -139,7 +139,7 @@
 	pinctrl-0 = <&spi_0_pins>;
 	pinctrl-names = "default";
 	status = "okay";
-	cs-gpios = <&tlmm 54 GPIO_ACTIVE_HIGH>;
+	cs-gpios = <&tlmm 54 GPIO_ACTIVE_LOW>;
```

Comment:
On which ipq40xx deice was this tested? 
Are you sure there is no additional GPIO polarity invert somewhere in the hip or the board?